musit-service/src/main/scala/no/uio/musit/service/NoCacheFilter.scala

```diff
@@ -0,0 +1,26 @@
+package no.uio.musit.service
+
+import javax.inject.Inject
+
+import akka.stream.Materializer
+import play.api.http.{HeaderNames, Status}
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
Should not use the scala default ExecutionContext. Unfortunately, the simplest example on the play documentation pages are using it. Which kind of indicates its OK...which it is not.
Instead, you should always use `import play.api.libs.concurrent.Execution.Implicits.defaultContext` in Play, unless you need to use your own EC imply in certain parts of the code.